lib/Immediate/Immediate.cpp

```diff
@@ -293,6 +294,38 @@ int swift::RunImmediately(CompilerInstance &CI, const ProcessCmdLine &CmdLine,
   if (Context.hadError())
     return -1;
 
+  // Load libSwiftCore to setup process arguments.
+  //
+  // This must be done here, before any library loading has been done, to avoid
+  // racing with the static initializers in the standard library and user code.
```

Comment:
Well, the standard library's initializers will still run. But we really shouldn't *race* with those; let's assume they're sensible and execute linearly.